src/RA_Core.cpp

```diff
@@ -220,7 +219,7 @@ static void InitCommon(HWND hMainHWND, /*enum EmulatorID*/int nEmulatorID, const
     //////////////////////////////////////////////////////////////////////////
     //	Image rendering: Setup image factory and overlay
     ra::services::g_ImageRepository.Initialize();
-    g_AchievementOverlay.Initialize(g_hThisDLLInst);
+    g_AchievementOverlay = AchievementOverlay{};
```

Comment:
The global will mostly be the same except the `ImageReference `members will actually have something this time.